docs/API.md

```diff
@@ -481,6 +481,29 @@ formatted as IPv4-mapped IPv6 addresses). By default (-1) this option is not set
 and the platform default value is used.
 ---
 
+| OptName               | Since | Binding | Type      | Units  | Default  | Range  |
+| --------------------- | ----- | ------- | --------- | ------ | -------- | ------ |
+| `SRTO_KMREFRESHRATE`  | 1.3.2 | pre     | `int32_t` | pkts   | 0x1000000| 0..unlimited |
+
+- **[GET or SET]** - number of packets to be transmitted after which the encryption
+key will be switched to the new one. The process involves "preannounce" when the new
+key is sent to the receiver, and "decommission" when the old key is no longer used;
+this extra period is customized with `SRTO_KMPREANNOUNCE` option.
+---
+
+| OptName               | Since | Binding | Type      | Units  | Default  | Range  |
+| --------------------- | ----- | ------- | --------- | ------ | -------- | ------ |
+| `SRTO_KMPREANNOUNCE`  | 1.3.2 | pre     | `int32_t` | pkts   | 0x1000 | see below |
+
+- **[GET or SET]** - number of packets to be transmitted between new key preannounce
```

Comment:
- **[GET or SET]** - The number of packets to be transmitted between the start of the preannounce period and the point where the Stream Encrypting Key (SEK) is switched. The allowed range for this value is between 1 and half of the current value of `SRTO_KMREFRESHRATE`.

Reply:
I copied the exact text, but I can see some small problem with formatting. The initial portion starts with a bullet, but then continued paragraph start unindented. Not sure how to fix it.

Reply:
- **[GET or SET]** - The interval (defined in packets) between when a new Stream Encrypting Key (SEK) is sent and when switchover occurs. This value also applies to the subsequent interval between when switchover occurs and when the old SEK is decommissioned.
At `SRTO_KMPREANNOUNCE` packets before switchover the new key is sent (repeatedly, if necessary, until it is confirmed by the receiver).
<br><br>
At the switchover point (see `SRTO_KMREFRESHRATE`), the sender starts encrypting and sending packets using the new key. The old key persists in case it is needed to decrypt packets that were in the flight window, or retransmitted packets.
<br><br>
The old key is decommissioned at `SRTO_KMPREANNOUNCE` packets after switchover .
<br><br>
The allowed range for this value is between 1 and half of the current value of `SRTO_KMREFRESHRATE`. The minimum value should never be less than the flight window (i.e. the number of packets that have already left the sender but have not yet arrived at the receiver).